### ISSUE TITLE
No extension logs displayed

### ISSUE THREAD

**Issue by AuHau** [raycast/extensions#1224]
Raycast version: 1.31.0

## Description

I wanted to debug Home Assistant extension and see its logs but when I ran `npm run dev` the extension was successfully loaded into Raycast, but I did not manage to get any logs in the terminal window. Here are the logs:

```
> homeassistant@1.2.4 dev
> ray develop

info  - entry points [src/index.tsx src/covers.tsx src/lights.tsx src/persons.tsx src/sensors.tsx src/binarysensors.tsx src/mediaplayers.tsx src/automations.tsx src/dashboard.tsx src/climate.tsx src/attributes.tsx src/batteries.tsx src/motions.tsx src/vacuums.tsx src/cameras.tsx src/scripts.tsx src/buttons.tsx src/scenes.tsx src/switches.tsx]
log: Must be admin to run 'stream' command
event - stopped development mode
```

I am wondering if the `log: Must be admin to run 'stream' command` might be related?

Not sure what other info you need about my setup. Happy to help debug it.

**Comment by fe9lix** [comment]
Hey @AuHau – does the same happen when you run `npm run dev` (instead of ray develop)? If that leads to the same error, you could check whether `sudo npm run dev` works; however, chances are something is off with the npm setup on your machine in that case since you shouldn't require admin rights. Did you install node and npm through [nvm](https://github.com/nvm-sh/nvm)?

**Comment by AuHau** [comment]
I have run `npm run dev` without a change and also the same with `sudo` without a change. Yes, I have my Node installed through `nvm` (Node v16.14.0)

**Comment by fe9lix** [comment]
Ok, so in the CLI, we use the system `log stream` command log under the hood to capture and print the log stream during dev mode. It looks like your login user does not have admin rights? If it's a managed machine, you could ask for getting access to the sys command under `/usr/bin/log` (there's a sys config file where sudo "exceptions" can be configured).

**Comment by AuHau** [comment]
Yeah that will be most probably the reason. My day-to-day account is a non-admin user because of security reasons. But I don't understand why it did not work when I have used `sudo` 🤔  I will try to work around this, but it is not my priority atm as I am not really working on any extensions atm. Feel free to close this as it seems like this is my local issue.
